### ISSUE TITLE
[1.20.1][Feature Request] Fluids have fluid rendering instead of solid color

### ISSUE THREAD

**Issue by Epb7304** [lorddusk/HQM#689]
Issue: fluids are displaying solid color textures
![image](https://github.com/lorddusk/HQM/assets/54673449/948a8903-0744-4683-b438-d48ba3b43370)

Change proposed: change back to old fluid rendering

OLD POST CONTENT:

This happens to modded fluids as well.

Specs:
Minecraft version: 1.20.1
mod version: HQM-1.20.1-5.16.0-forge
architectury version: architectury-9.1.12-forge
forge version: NeoForge 47.1.100
operating system: Windows 10
Windows OS build: 19045.4046

no other mods were installed at the time, however, the issue persists with other mods installed.

Reproducibility:
step 1: download mods and run Minecraft NeoForge using the specs given above.
step 2: create a world, enable questing and editing, and give yourself a book
step 3: create a set and quest, then attempt to change the icon of that quest to a fluid (not a bucket)

**Comment by canitzp** [comment]
Can you please tell me which items do use the barrier textures? I have tested a consume task, with fluids as source and yes there coloring is odd, but that is intentional. Therefor the water and lava (first 2 entries) of your quest are correct

**Comment by Epb7304** [comment]
wait so the fluids are supposed to be solid colored textures? In jei it shows the correct textures

![image](https://github.com/lorddusk/HQM/assets/54673449/029e8638-f603-4dd7-904f-bbd2a45a9922) 
![image](https://github.com/lorddusk/HQM/assets/54673449/19067929-6468-429e-8596-72e0647abe28)

in previous Minecraft modpacks that used an earlier version of this mod (Agrarian skies) it shows correctly

![image](https://github.com/lorddusk/HQM/assets/54673449/a7cfdf7f-c651-4918-a075-f540a7616b22)
![image](https://github.com/lorddusk/HQM/assets/54673449/d3bf906f-63c2-432a-ac50-2a1581086a96)

also worth noting that a few mods that I also have installed add liquids themselves
they all show various solid colors, and not the textures they should have, they also modify other fluid textures in the book, so water appears jet black
![image](https://github.com/lorddusk/HQM/assets/54673449/58cac114-682b-441e-95ba-551a7b5cbfd2)

**Comment by canitzp** [comment]
Yes the single color is currently intended. In a previous version, the fluid was rendered correctly, but one of the many mc/forge updated, changed the fluid rendering and to simplify the porting someone just used a single color for representation. This can be changed back to fluid rendering.

**Comment by Epb7304** [comment]
Ah, I'm guessing that was probably around 1.13 since it had a lot of back-end changes. Well I'll edit the title to reflect that it is now a request

**Comment by Epb7304** [comment]
by the way, this is what it looks like when you have ~10 mods that add various fluids. it becomes confusing very quickly
![image](https://github.com/lorddusk/HQM/assets/54673449/0813a7e0-379c-4c49-a0d0-d8684c41e76e)
Description: from top left to bottom right
- Water, Lava, Witch Water, Sea Water, Honey, Milk, Essence of Knowledge
- Honey (but different ig) Potion, Destabilized Redstone, Energized Glowstone, Resonant Ender, Sap, Syrup
- Resin, Tree Oil, Latex, Creosote Oil, Crude Oil. Heavy Oil, Light Oil
- Refined Fuel, life_essence_fluid_type, doubt_fluid_type, fluid_type.thaumia.liquid_quicksilver, Latex (but different?), Liquid Meat, Sewage
- Essence, Sludge, Pink Slime, Biofuel, Ether Gas, Raw Ore Meat, Fermented Ore Meat

The latexs are different mods (thermal and industrial foregoing)
The honey are different mods as well (cofh_core and productive bees)

This is without any mods that add a ton of liquids link tinkers
